modules/search/search.go

```diff
@@ -0,0 +1,166 @@
+package search
+
+import (
+	"github.com/bwmarrin/discordgo"
+	"github.com/lordralex/absol/api"
+	"github.com/lordralex/absol/api/database"
+	"github.com/lordralex/absol/api/logger"
+	"github.com/spf13/viper"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type Module struct {
+	api.Module
+}
+
+// Load absol commands API
+func (*Module) Load(ds *discordgo.Session) {
+	api.RegisterCommand("s", RunCommand)
+	api.RegisterCommand("search", RunCommand)
+
+	api.RegisterIntentNeed(discordgo.IntentsGuildMessages, discordgo.IntentsDirectMessages)
+}
+
+func RunCommand(ds *discordgo.Session, mc *discordgo.MessageCreate, _ string, args []string) {
+	if len(args) == 0 {
```

Comment:
I am going to require this only works in DMs. I do not want any channel to have this clutter.

Reply:
Should i do this with both commands, or only the search command?

Reply:
MCPing is okay to not be in DMs. Search must be done via DMs, in that you can only trigger it in DMs.